docs/troubleshooting-guide/application-database.md

```diff
@@ -1,36 +1,43 @@
-# Trouble with the Metabase Application Database
+# The application database isn't working
 
-<div class='doc-toc' markdown=1>
-- [Metabase fails to start due to database locks](#database-locks)
-- [Metabase H2 application database gets corrupted](#h2-app-db-corrupted)
-- [Metabase fails to connect to H2 Database on Windows 10](#h2-connect-windows-10)
-</div>
+You have installed Metabase, but:
 
-Metabase stores information about users, questions, and so on in a database of its own that we call the "application database". The default application database that ships with Metabase is an [H2 database][what-is-h2], which is not recommended for production. To use a production-ready database for you Metabase application database, see [Migrating from H2][migrating]. 
+- it logs a `liquibase` error message when you try to run it,
+- it logs another error message that mentions `H2` or `h2` while it is running, or
+- you are on Windows 10 and get a warning about file permissions.
 
-<h2 id="database-locks">Metabase fails to start due to database locks</h2>
+## Are you using H2 as your application database?
 
-Sometimes Metabase will fail to start up because a database lock did not clear properly. 
+**Root cause:** Metabase stores information about users, questions, and so on in a database of its own called the "application database", or "app database" for short. By default Metabase uses [H2][what-is-h2] for the app database, but we don't recommended it for production---because it's an on-disk database, it's sensitive to filesystem errors, such as a drive being corrupted or a file not being flushed properly.
 
-**How to detect this:** The error message will look something like:
+**Steps to take:**
+
+1. FIXME how to tell if you're using H2
```

Comment:
Admin > Troubleshooting > "Diagnostic Info" - scroll down to `application-database`

Reply:
added - thanks